atrium/AtriumClient.php

```diff
@@ -47,13 +47,13 @@ function updateUser($userGUID, $params) {
 
     $json = json_encode($user);
 
-    $response = $this->makeRequest('PUT', '/users/' . $userGUID, $json);
-    return new User(json_decode($response, true)['user']);
+    $response = $this->makeRequest('PUT', "/users/$userGUID", $json);
+    return new User($response['user']);
   }
 
   function listUsers() {
     $response = $this->makeRequest('GET', '/users', '');
```

Comment:
What is the third `''` argument to `makeRequest` for?

Reply:
Looking at the code, seems like this is for the body. You can assign it a default argument of `''` so you don't have to pass it every time.

```php
private function makeRequest($method, $endpoint, $body = "") {
```

Reply:
Great catch. For some reason I originally thought default args were not supported in PHP. Updated 👍